main.java

```diff
@@ -7,9 +7,15 @@ public static void main(String[] args) {
 	int z = 1;
 	int x = 2;
 	System.out.println("Sum of " + z + " and " + x + " equals " + sum(z, x) + ".");
+	System.out.println("2 * 2 = " + mult(2, 2));
+
     }
     
     public static int sum(int a, int b) {
     	return a + b;
     }
-}
+
+    public static int mult(int a, int b){
```

Comment:
Add a space before the brace.
before: public static int mult(int a, int b){
after:    public static int mult(int a, int b) {